documentation/1.9/content/samples/wtc-model.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Modeling a WTC configuration"
+date: 2021-11-01T10:19:24-05:00
+draft: false
+weight: 3
+description: "A domain model with a typical queue based configuration for a WebLogic Tuxedo Connector."
+---
+
+This WDT domain model sample section shows a remote and local queue for a WTC configuration. 
+
+```yaml
+ resources:
+   StartupClass:
+     WtcServerSideHelper:
+       Target: admin
+       ClassName: wlstest.functional.core.wtc.conversation.common.apps.Helper.WtcServerSideHelperImpl
+   WTCServer:
+     myWTCServer:
+       Target: admin
+       WTCExport:
+         'WTCExportedService-1':
+           ResourceName: QaWls2Conv2
```

Comment:
Please call out for the reader what each of these fields maps to.  The names are not descriptive enough to be self-explanatory.

Reply:
Maybe as # comments?